.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
it's better to make a new commit with this
as this one is not related to "Divide scheduler module"